src/StaticWebAssetsSdk/Tasks/ApplyCompressionNegotiation.cs

```diff
@@ -27,55 +27,33 @@ public override bool Execute()
             .GroupBy(e => e.AssetFile)
             .ToDictionary(g => g.Key, g => g.ToList());
 
-        var compressedAssets = assetsById.Values.Where(a => a.AssetTraitName == "Content-Encoding").ToList();
         var updatedEndpoints = new HashSet<StaticWebAssetEndpoint>(StaticWebAssetEndpoint.RouteAndAssetComparer);
 
-        var preservedEndpoints = new Dictionary<(string, string), StaticWebAssetEndpoint>();
```

Comment:
This was only used to capture additional endpoints that we needed to put into the UpdatedEndpoints group. Since `updatedEndpoints` is a `HashSet` we can directly add the other endpoints there without worry